src/rules/extensions.ts

```diff
@@ -43,11 +61,21 @@ export interface OptionsItemWithPatternProperty {
   ignorePackages?: boolean
   checkTypeImports?: boolean
   pattern: ModifierByFileExtension
+  fix?: boolean
```

Comment:
Some kind of related to #230, `fix` is a bad name, let's use `forceFix` instead and use `suggestions` when `forceFix` is falsy.